breakout_3_Observability-with-AI/breakout_3_exercise.md

```diff
@@ -0,0 +1,50 @@
+## Part 3 - Observability with AI
+#### How to find the same root cause with your friendly neighborhood Grafana Assistant
+For this exercise you will work through troubleshooting scenarios from the previous sections, however this time using AI tools, specifically [Grafana Assistant](https://grafana.com/docs/grafana-cloud/machine-learning/assistant/), to find the solution.
+
+### Instructions
+Go to the URL in the slides where you will have access to the Grafana Cloud Stack with:
+1. Frontend O11y
+1. App O11y
+1. K8s Monitoring
+
+Navigate to the indicated app in each section, then click on the Grafana Assistant button in the upper right hand corner to get started. The same prompt window can and should be used for a section for context and question continuity.
+![grafana-assistant](/images/breakout_3/0.1-grafana-assistant.png)
+
+You will have 10 minutes to work through the questions, after which we will regroup to work through everything together for the final 10 minutes.
+
+Answers and guidance on how to get to the answer are in the [breakout_3_answers](./breakout_3_answers) folder, however we encourage you to try to work through the questions on your own first!
+
+___NOTE___: Due to the stochastic nature of an LLM assistant, the results will not always look the same between users or match what we found in previous exercises. 
+
+### Questions
+**Query Frontend O11y Data**
```

Comment:
I struggled to get this to use logs instead of prometheus metrics leading to this not giving correct answers - how did you give it the context?

Reply:
@Alex3k Check out the answer for 1.4-grafana-assistant, I showed how I re-prompted to get it working a bit better, let me know if there is a better way to go about this.
https://github.com/grafana/observability-workshop-breakouts/blob/obscon-updates/breakout_3_Observability-with-AI/breakout_3_answers/1.4-grafana-assistant.md

Reply:
I think we just need to add a caveat to say that the engineering team are working on a specialist tool for frontend o11y and explain there may be issues with it today. Let's call a spade a spade :)